packages/web-components/src/tabs/tabs.ts

```diff
@@ -0,0 +1,194 @@
+import { attr } from '@microsoft/fast-element';
+import { FASTTabs } from '@microsoft/fast-foundation';
+import { Tab } from '../index.js';
+import { TabsAppearance, TabsSize } from './tabs.options.js';
+
+export const TabTokenNames = {
+  tabIndicatorOffset: '--tabIndicatorOffsetX',
```

Comment:
DO we want these to be tokens or do we just want to assign css variables and use the `$fastController.addStyles`  @chrisdholt for his opinion.

Reply:
@chrisdholt do you have any comment here?